expression_tools/put_in_dir.cwl

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env cwl-runner
+# originally from https://github.com/mskcc/pluto-cwl
+
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+class: ExpressionTool
+# class: CommandLineTool
+id: put-in-dir
+
+inputs:
+  output_directory_name: string
+  files:
+    type:
+      type: array
+      items:
+        - File
```

Comment:
I'm not sure what will happen if this tool tries to put `string`s and `'null'`s into a Directory, it might fail, or create empty files with "string" and "null" as the filenames. I'd limit this to just accepting files and Directories

Reply:
just tested it. giving it a string causes it to fail. giving it `null` is fine though